monitoring/monitorlib/auth.py

```diff
@@ -313,75 +313,6 @@ def issue_token(self, intended_audience: str, scopes: List[str]) -> str:
         }
         payload = "&".join([k + "=" + v for k, v in query.items()])
 
-        # Generate signature
```

Comment:
It isn't valid to simply remove this block of code -- as currently proposed:

1. we're left with a number of SignedRequest class parameters that don't do anything and therefore also have incorrect documentation (e.g., signature_style)
2. message_signing isn't added as a parameter to the SignedRequest constructor which would pass it to the AuthAdapter parent class (or the existing parameters used to create an appropriate message_signing argument)
3. SignedRequest is no longer a valid class name for what this class does
4. dead signing code is not removed
5. there is no graceful deprecation of the now-lost capability to sign token requests in the UPP2 format, nor is there a comment in the PR that this capability is being removed

I'd recommend against trying combine SignedRequest signing removal into this PR as it's a non-trivial amount of effort and not integral to what the rest of the PR is trying to accomplish.